File: c++/writeToPipe.cpp
#include <iostream>
#include <fcntl.h>
#include <unistd.h>
#include <stdio.h>
#include <string.h>

int main(){
	char FIFO[] = "../mypipe";
	int fd;
	while (1) {
		fd = open(FIFO, O_WRONLY);
		write(fd, "1", 1);
		close(fd);
	}
	return 0;
}

File: writeToPipe.py
import posix
import os

FIFO = 'mypipe'
#os.mkfifo(FIFO)

def writeToPipe():
	while True:
		with open (FIFO, 'wb', 0) as file:
			file.write(b'hi')
		print("Hallelujah")

writeToPipe()